vcd/resource_vcd_nsxt_edgegateway_rate_limiting.go

```diff
@@ -161,7 +139,7 @@ func resourceVcdNsxtEdgegatewayRateLimitingDelete(ctx context.Context, d *schema
 
 // resourceVcdNsxtEdgegatewayRateLimitingImport imports rate limiting (QoS) configuration for NSX-T
 // Edge Gateway.
-// The import path for this resource is Edge Gateway. ID of the field is also Edge Gateway ID as it
+// The import path for this resource is Edge Gateway. ID of the field is also Edge Gateway ID as
```

Comment:
Is the comment incomplete? _"is also Edge Gateway ID as..."_
cc: @Didainius -> as I think you implemented the import of the rate limiting.

Reply:
The comment's other side should be on the next line, just to keep the lines shorter and more readable

Reply:
Ah, sorry, I was using the split view and I didn't see it 😇